utils/ort/src/main/kotlin/CopyrightStatementsProcessor.kt

```diff
@@ -256,6 +187,32 @@ private fun replaceYears(copyrightStatement: String): Pair<String, Set<Int>> {
  * TODO: Maybe treat URLs similar to years, e.g. entries which differ only in URLs and years can be merged.
  */
 object CopyrightStatementsProcessor {
+    data class Parts(
+        val prefix: String,
+        val years: Set<Int>,
+        val owner: String,
+        val originalStatements: List<String>
+    ) {
+        companion object {
+            val COMPARATOR = compareBy<Parts>({ it.owner }, { prettyPrintYears(it.years) }, { it.prefix })
```

Comment:
Should this comparator now become private, as it is an internal of `CopyrightStatementsProcessor` ?

Reply:
If you don't mind, I'd prefer to do also that in a later PR as I'm planning to implement `Comparable` here.